lmnet/lmnet/networks/object_detection/yolo_v2.py

```diff
@@ -1253,27 +1262,30 @@ def __calculate_truth_and_maskes(
         sum_conf = 0.0
         sum_diff_conf = 0.0
 
+        # extra coordinate loss for early training steps to encourage predictions to match anchor.
+        # https://github.com/pjreddie/darknet/blob/2f212a47425b2e1002c7c8a20e139fe0da7489b5/src/region_layer.c#L248
+        if self.seen < self.seen_threshold:
+            if self.is_debug:
+                print("current seen: {}. To calculate coordinate loss for early training step.".format(self.seen))
+
+            offset_x, offset_y, offset_w, offset_h = YoloV2.py_offset_boxes(num_cell_y, num_cell_x,
+                                                                            self.batch_size, self.boxes_per_cell,
+                                                                            self.anchors)
+
+            stride_x = image_size[1] / num_cell_x
+            stride_y = image_size[0] / num_cell_y
+
+            cell_gt_boxes[:, :, :, :, 0] = (offset_x + 0.5) * stride_x
+            cell_gt_boxes[:, :, :, :, 1] = (offset_y + 0.5) * stride_y
+            cell_gt_boxes[:, :, :, :, 2] = stride_x * offset_w
+            cell_gt_boxes[:, :, :, :, 3] = stride_y * offset_h
+            cell_gt_boxes[:, :, :, :, 4] = -1
+
+            coordinate_maskes += 1  # True
```

Comment:
```suggestion
            coordinate_maskes[:] = 1  # True
```

`+=` force to confirm `coordinate_maskes` before values.